### DISCUSSION TITLE
VideoDownloadHelper only downloads partial youtube videos

### DISCUSSION THREAD

**Discussion by ElCaliente42** [aclap-dev/video-downloadhelper#376] [Help and Q&A]
### Your question

Hi, just paid for another license as the previous license no longer
worked with Firefox 124.0.2 64-bit for Windows. on Windows 11 pro 64 bit v23H2

However, this license only downloads partial videos from youtube -
for instance only 16 minutes of a 1.5 hr movie or 5 minutes if I select
MKV.

Note, this was all working fine a few days ago, and it seems to work OK with other services (GAAGo for instance).

here's a sample link:
https://www.youtube.com/watch?v=Bx-F6630QFE&t=132s

I tested on a different machine with Windows 11 Enterprise and it seems it works there OK using the same license and same youtube account and same video.
.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 02ceb725b8944ba1
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\raule",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
looks like there are limitations, not available in my country

**Comment by ElCaliente42** [comment]
I tried several other videos with the same result.

**Comment by ElCaliente42** [comment]
Here are a couple of others:
https://www.youtube.com/watch?v=Z8XgBs6E3W4

https://www.youtube.com/watch?v=7_1iEoCILuQ&t=18s

**Reply by jcfrog** [reply]
the first one is not available for me, the second one downloaded fine.

**Comment by hfball** [comment]
This seems to have cropped up again. Downloads stall out and terminate. The best I got was 10 minutes of a 26 minute video.

The download speed is far faster than it would need to be to stream it, and then Google seems to recognize this and cuts it off.
